File: 2022-1/ALGORITMOS/TDE_26:05/ex2.c
#include <stdio.h>

int main(){
    int valor[6],i;
    for(i=0;i<6;i++){
        printf("Digite os valores:\n");
        scanf("%d%*c",&valor[i]);
    }
    for(i=0;i<6;i++){
        printf("valor digitado:%d\n",valor[i]);
    }
}

File: 2023-2/Estrutura de Dados/aula3/ex2.py
numeros_validos = []
entrada = input("Digite números inteiros separados por espaços: ").split()
for item in entrada:
    try:
        numero = float (item)
        numeros_validos.append(numero)
    except ValueError:
        print(f"Valor inválido: (item). Ignorando este valor.")
    print(f"Os valores válidos digitados foram: {numeros_validos}")

    soma = 0
    for numero in numeros_validos:
        soma += numero
    print(f"A soma dos valores válidos é: {soma}")